app/main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+from typing import List
+
+
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills: List[str] = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills: List[str] = ["JavaScript", "HTML", "CSS"]
```

Comment:
Do not redefine the list created in the `SoftwareEngineer` class, extend it with new values using `+=` or `extend()`. Apply this change to all classes